source/Scenes/Components/ExitMenuOverlay/RefutationButton.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Annex.Graphics.Events;
+using Annex.Scenes.Components;
+using System;
+
+namespace Game.Scenes.Components.ExitMenuOverlay
+{
+    public class RefutationButton : Button
+    {
+        public Action OnClickHandler { get; set; }
+
+        public RefutationButton(string Id) : base(Id)
+        {
+        }
+
+        public override void HandleMouseButtonReleased(MouseButtonReleasedEvent e)
```

Comment:
I'd probably change this to Pressed and not Released - If you hold down a mousebutton outside of the button, then move your mouse to the button while holding, and then release, you'll trigger the HandleMouseButtonReleased event.